.gitignore

```diff
@@ -20,7 +20,7 @@
 x64/
 x86/
 bld/
-[Bb]in/
+# [Bb]in/
```

Comment:
IMPORTANT: make sure this includes `chef-powershell/bin` and that all of those DLLs are removed from this PR.

Reply:
gotcha. Done